src/NuGet.Services.Staging.BackgroundWorkers/MessageListener/TopicMessageListener.cs

```diff
@@ -40,7 +41,10 @@ public TopicMessageListener(IOptions<TopicMessageListenerOptions> options, ILogg
             _options = options.Value;
             _logger = logger;
 
-            var factory = MessagingFactory.CreateFromConnectionString(_options.ServiceBusConnectionString);
+            var factory = MessagingFactory.Create(
+                _options.ServiceBusUri,

```

Comment:
Changed the way service bus topic subscription is accessed. The previous code forced me to use a SAS token of the entire service bus, and this way the SAS token is per topic (safer).